test/e2e/tests/base_tag.spec.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+describe('SCE URL policy when base tags are present', function() {
+  function checkUrl(url, allowed) {
+    var urlIsTrusted = browser.executeScript('return isTrustedUrl(arguments[0])', url);
+    expect(urlIsTrusted).toBe(allowed);
+  }
+
+  beforeAll(function() {
+    loadFixture('base_tag');
```

Comment:
Can you change all names to kebab-case (to be consistent with the rest of the tests)?